tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
+
         // Carbon::parse('2018-01-05 00:00:00')->addDays(6)->calendar(Carbon::parse('2018-01-05 00:00:00'))
+        // 'Thursday at 12:00 AM'
         'ketvirtadienis at 00:00',
+
         // Carbon::parse('2018-01-06 00:00:00')->addDays(6)->calendar(Carbon::parse('2018-01-06 00:00:00'))
+        // 'Friday at 12:00 AM'
         'penktadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(2)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Tuesday at 12:00 AM'
         'antradienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(3)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Wednesday at 12:00 AM'
         'trečiadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(4)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Thursday at 12:00 AM'
         'ketvirtadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(5)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Friday at 12:00 AM'
         'penktadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(6)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Saturday at 12:00 AM'
         'šeštadienis at 00:00',
+
         // Carbon::now()->subDays(2)->calendar()
+        // 'Last Sunday at 8:49 PM'
         'Last sekmadienis at 20:49',
+
         // Carbon::parse('2018-01-04 00:00:00')->subHours(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Yesterday at 10:00 PM'
         'Yesterday at 22:00',
+
         // Carbon::parse('2018-01-04 12:00:00')->subHours(2)->calendar(Carbon::parse('2018-01-04 12:00:00'))
+        // 'Today at 10:00 AM'
         'Today at 10:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->addHours(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Today at 2:00 AM'
         'Today at 02:00',
+
         // Carbon::parse('2018-01-04 23:00:00')->addHours(2)->calendar(Carbon::parse('2018-01-04 23:00:00'))
+        // 'Tomorrow at 1:00 AM'
         'Tomorrow at 01:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->addDays(2)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Tuesday at 12:00 AM'
         'antradienis at 00:00',
+
         // Carbon::parse('2018-01-08 00:00:00')->subDay()->calendar(Carbon::parse('2018-01-08 00:00:00'))
+        // 'Yesterday at 12:00 AM'
         'Yesterday at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(1)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Yesterday at 12:00 AM'
         'Yesterday at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Tuesday at 12:00 AM'
         'Last antradienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(3)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Monday at 12:00 AM'
         'Last pirmadienis at 00:00',
```

Comment:
`Paskutinį pirmadienį 00:00` and so on

`Last` is directly translated to `Paskutinį`. Nominative form would be `Paskutinis`

Reply:
Updated.